CHANGELOG.md

```diff
@@ -4,7 +4,14 @@ This file documents the version history of OpenDP. The links on each version num
 showing the source changes from the previous version.
 
 
-## [0.12.1-dev](https://github.com/opendp/opendp/compare/v0.12.0...HEAD) - TBD
+## [0.12.2-dev](https://github.com/opendp/opendp/compare/v0.12.1...HEAD) - TBD
```

Comment:
The version bumps might go beyond a metadata fix, but if they are the right thing to do, great!

Reply:
I did it to be able to demo the fix.